### PR TITLE
Add/remove user from team fails on second add #528

### PR DESCRIPTION
**Pull request by andrejonathan**
Discovered that the openshiftResource had array values that were returning as null if the array was empty in OpenShiftApiPolicy. Added a fix that sets the arrays to [] if null.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@atomist/cli": "^1.0.2",
-    "@atomist/automation-client": "1.1.0-master.20181121142425",
+    "@atomist/automation-client": "1.1.0-master.20181121110216",
```
Comment (kbristow) [inline package.json]: This kills the quantum-mechanic.

File: src/openshift/api/OpenShiftApiPolicy.ts [hunk 2]
```diff
@@ -13,6 +13,13 @@ export class OpenShiftApiPolicy extends OpenShiftApiElement {
         const roleBindingResourceObject = await this.getRoleBindingResource(role, namespace);
         const openshiftRole = roleBindingResourceObject.roleBinding;
 
+        openshiftRole.userNames == null ? openshiftRole.userNames = [] : openshiftRole.userNames = openshiftRole.userNames;
+        openshiftRole.subjects == null ? openshiftRole.subjects = [] : openshiftRole.subjects = openshiftRole.subjects;
+
+        if (openshiftRole.userNames == null || openshiftRole.subjects == null ) {
+            logger.debug(`nulls openshiftRole = ${openshiftRole}`);
```
Comment (kbristow) [inline src/openshift/api/OpenShiftApiPolicy.ts]: Do you still want to leave this in here?

File: package.json (hunk 1 above)
Comment (andrejonathan) [reply package.json]: Fixed thanks

File: src/openshift/api/OpenShiftApiPolicy.ts (hunk 2 above)
Comment (andrejonathan) [reply src/openshift/api/OpenShiftApiPolicy.ts]: Fixed thanks

**Pull request merged**